### ISSUE TITLE
npm warnings

### ISSUE THREAD

**Issue by mikeharder** [aspnet/JavaScriptServices#1051]
# Repro Steps
1. `dotnet new [angular|react]`
2. `npm install`

# Expected
No warnings

# Actual
```
npm WARN angular@0.0.0 No description
npm WARN angular@0.0.0 No repository field.
npm WARN angular@0.0.0 No license field.
```

# Dependency Versions
CLI: https://dotnetcli.azureedge.net/dotnet/Sdk/2.0.0-preview2-006494/dotnet-sdk-2.0.0-preview2-006494-win-x64.zip
NodeJS: 8.1.2

**Comment by rosieks** [comment]
It's duplicate of #897

**Comment by SteveSandersonMS** [comment]
As @rosieks indicated, this was fixed when #1044 was merged, but it's too late for that commit to make it into 2.0.0-preview2 unless we decide it's really important. I think it's very unlikely that fixing this would be essential for preview2 at this stage (it's very unlikely to trouble anyone), but please let me know if you disagree!

Closing since it's already fixed in `dev`.

**Comment by mikeharder** [comment]
I agree it's fine to fix in dev but not preview2.
